### ISSUE TITLE
add shell extension

### ISSUE THREAD

**Issue by Hope-IT-Works** [Hope-IT-Works/PS2EXE-GUI#2]
**Is your feature request related to a problem? Please describe.**
you need to start the converter first, why don't just right-click and convert on the go?

**Describe the solution you'd like**
When right-clicking a `.ps1`-file, allow the user to directly convert this script via PS2EXE.

**Example**
![image](https://user-images.githubusercontent.com/52013820/190667031-8e078fcc-d232-433e-9300-0552a436d491.png)

**Comment by couleurm** [comment]
yeah that can be done with a few reg keys, you can assign it the logo of powershell and the args to it

great tool for the job: https://github.com/BluePointLilac/ContextMenuManager

**Comment by Hope-IT-Works** [comment]
sounds great! need to investigate which registry keys are needed exactly, so that the executable could create the registry keys automatically.

have a nice weekend!

**Comment by steve02081504** [comment]
https://github.com/steve02081504/ps12exe/blob/v0.1.30/src/ContextMenuAdder.ps1
hope this helps

**Comment by steve02081504** [comment]
> @steve02081504 Thanks! Gonna take a look at that soon. Just saw your version of ps2exe and now I'm thinking about which one is better. Could you summarize what your project does differently? Maybe we could work together, as I'm just creating a GUI with WPF.

Personally I think my version of the repo is better (otherwise I wouldn't have had to risk getting blocked by the original project owner to fork it would I)
I think the only downside might be that it's no longer a single file of ps code, [other comparisons can be seen here](https://github.com/steve02081504/ps12exe?tab=readme-ov-file#comparative-advantages-)
As well as sorry that ps12exe already has [a built-in gui](https://github.com/steve02081504/ps12exe/blob/master/docs/README_EN.md#gui-parameters), but I guess our collaboration can be done via pr if you want.
